### PR TITLE
[TASK] config: Move config.js for easier deployments

### PR DESCRIPTION
**Pull request by MyIgel**
## Description
Moved the `config.js` file to `config.ffrgb.js` and add `config.js` to`.gitignore` to allow customization.

## Motivation and Context
To deploy the map to a server it is required to customize the `config.js` file for every community other than ffrgb. Currently that means that this file gets resettet every time it changes upstream and/or at least cause merge conflicts.
To prevent these problems the `config.js` should be added to `.gitignore` so it does not get affected. The original `config.ffrgb.js` can be used as a reference and/or template which could be copied before the build step in the ffrgb deployment.

At the end this small change would be a great improvement for any other community using the meshviewer and only requires one more `cp` command to be run before a build of the ffrgb maps `gulp` call.

## How Has This Been Tested?
As it only affects the configuration file no further tests are needed.

## Screenshots/links (if appropriate):
`-`

## Checklist:
- [x] My code follows the code style of this project. (CI will test it anyway and also needs approval)
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly, see ffrgb/meshviewer-doc#9

### CODE REVIEW

**Comment by MyIgel** [comment]
How can I fix the `netlify` deployment? It seems that it has to be done in their configuration? Imho that should not be fixed while running gulp ;)

**Comment by xf-** [comment]
Like always - i want a running version in git.

You can use cp for your config and ignore the config
```git
git update-index --assume-unchanged config.js
```

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,17 @@
+# Generic temp files
```
Comment (xf-) [inline .gitignore:1]: This list is endless. Developers should use global gitignore config for generic stuff like that.
http://egorsmirnov.me/2015/05/04/global-gitignore-file.html

**Comment by MyIgel** [comment]
Äh what???
How about any discussion and/or the possibility to fix it? Just closing PR's because anyone dont did something the way you want it to be isn't that nice... :(

**Comment by Grotax** [comment]
My recommendation: fork the repository for your community and use your fork for deployment.
This way other people can step in if you no longer want to provide the map for your community.  Just create on commit with your adjustments and rebase it on upstream.

**Comment by MyIgel** [comment]
Thats an even more hacky workaround (and would require a force push on every change), the better option would be to use the ffrgb config as a fallback if no `config.js` file was provided?

**Comment by Grotax** [comment]
Would be fine by me. That would also provide a running version in git.

**Pull request closed without merging**